### PR TITLE
1672 rename deviceid to androidid

### LINKED ISSUE #1672: Rename from `deviceId` to `androidId`
To make the code in the `Device` entity consistent with the code in the Analytics app, rename from `Device#deviceId` to `Device#androidId`.

### PR DESCRIPTION
**Pull request by alexander-kuruvilla**
Closes #1672 

Changes:
Updated the attribute deviceId to androidId in Device Entity
Updated setters and getters and references.
Created database migration script for changing the value in database.

### CODE REVIEW

File: src/main/resources/db/migration/2004019.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# 2.4.19
```
Comment (jo-elimu) [inline src/main/resources/db/migration/2004019.sql:1]:
```suggestion
# 2.4.20
```

Comment (jo-elimu) [inline src/main/resources/db/migration/2004019.sql:1]: @alexander-kuruvilla The latest version now is 2.4.20, so you can rename the file to `2004020.sql`.

**Review by jo-elimu — APPROVED** [review]

**Review by jo-elimu — DISMISSED** [review]
Also the `2004019.sql` file itself needs to be renamed.

**Comment by alexander-kuruvilla** [comment]
> Also the `2004019.sql` file itself needs to be renamed.

Done. Changed to 2004020.sql

**Review by jo-elimu — APPROVED** [review]

**Pull request merged**